build.sbt

```diff
@@ -3,8 +3,8 @@ import com.typesafe.sbt.SbtGit._
 
 lazy val commonSettings = Util.settings ++ versionWithGit ++ Seq(
   organization := "org.scala-sbt",
-  git.baseVersion := "0.1.0-M1",
-  isSnapshot := false,
+  git.baseVersion := "0.1.0",
+  isSnapshot := true,
```

Comment:
Not sure what the right value should be, but I'm using isSnapshot to turn off git versioning in the lines below.